app/models/concerns/notifiable.rb

```diff
@@ -0,0 +1,35 @@
+module Notifiable
+  extend ActiveSupport::Concern
+
+  def notifiable_title
```

Comment:
I think just `title` could be good enough, like `notification.title` at [the only usage of `notificable_title`](https://github.com/consul/consul/blob/master/app/views/notifications/_notification.html.erb#L9), as it was a column in the Notifications table.